com.adsbynimbus.nimbus/Runtime/Plugins/iOS/NimbusManager.swift

```diff
@@ -182,6 +186,27 @@ import UnityAds
         }
     #endif
     
+    #if NIMBUS_ENABLE_MOBILEFUSE
+        @objc public class func initializeMobileFuse() {
+            Nimbus.shared.renderers[.mobileFuse] = NimbusMobileFuseAdRenderer()
+            let _ = NimbusMobileFuseRequestInterceptor()
+        }
+        @objc public class func fetchMobileFuseToken() -> String {
+            var tokenData: [String : String] = ["":""]
+            do {
+                let group = DispatchGroup()
+                group.wait(for: {tokenData = try await MobileFuseRequestBridge().tokenData})
+                guard let data = try? JSONEncoder().encode(tokenData),
+                let jsonString = String(data: data, encoding: .utf8) else {
+                       return "{}"
```

Comment:
In IOS.cs you are doing a check if the return from this method is `nullOrEmpty` to short circuit the request modifications and returning `{}` does not pass that check.